terraform/modules/task-definitions/signon/main.tf

```
@@ -0,0 +1,78 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "eu-west-1"
+
+  assume_role {
+    role_arn = var.assume_role_arn
+  }
+}
+
+data "aws_secretsmanager_secret" "sentry_dsn" {
+  name = "signon_app-SENTRY_DSN"
+}
+
+locals {
+  service_name = "signon"
+}
+
+module "task_definition" {
+  source             = "../../task-definition"
+  mesh_name          = var.mesh_name
+  service_name       = local.service_name
+  cpu                = 512
+  memory             = 1024
+  execution_role_arn = var.execution_role_arn
+  task_role_arn      = var.task_role_arn
+
+  container_definitions = [
+    {
+      "name" : "signon",
+      "image" : "govuk/signon:deployed-to-production",
+      "essential" : true,
+      "environment" : [
+        { "name" : "GOVUK_APP_NAME", "value" : "signon" },
+        { "name" : "GOVUK_APP_ROOT", "value" : "/var/apps/signon" },
+        { "name" : "PORT", "value" : "8080" },
+        { "name" : "DATABASE_URL", "value" : "${var.signon_db_url}" },
+        { "name" : "RAILS_ENV", "value" : "development" },
```

I think you'll almost certainly want the rails env to be "production".

```suggestion
        { "name" : "RAILS_ENV", "value" : "production" },
```